example_computer_setup/add_extras.py

```diff
@@ -34,11 +34,13 @@ def main(profile, config):
 
     echo.echo_success(f"Added the following properties to {label}: {extras}")
 
-    echo.echo_info(f"Setting the conda directory for computer {label}")
-    conda_dir = get_conda_dir(computer)
-    computer.set_property("conda_dir", conda_dir)
-    computer.store()
-    echo.echo_success(f"Set the Conda directory on {label} to '{conda_dir}'")
+    if not "conda_dir" in extras:
+        echo.echo_info(f"Setting the conda directory for computer {label}")
+        conda_dir = get_conda_dir(computer)
+        computer.set_property("conda_dir", conda_dir)
+        computer.store()
+
+        echo.echo_success(f"Set the Conda directory on {label} to '{conda_dir}'")
```

Comment:
IMO we should also check that the specified `conda_dir` exists on the computer.

I understand that this doesn't guarantee that it will still be there when a Calculation is run, but catching it early will prevent typos

Reply:
good point, done!

Reply:
I don't see any changes; were they pushed?